src/FastRouteRouterFactory.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive\Router;
+
+use Interop\Container\ContainerInterface;
+
+/**
+ * Create and return an instance of FastRouteRouter.
+ *
+ * Configuration should look like the following:
+ *
+ * <code>
+ * 'router' => [
+ *     'fastroute' => [
+ *         'cache_enabled' => true, // true|false
+ *         'cache_file'   => '(/absolute/)path/to/cache/file', // optional
+ *     ],
+ * ]
+ * </code>
+ */
+class FastRouteRouterFactory
```

Comment:
Yes, but is the factory needed at all? How would it be used in a standard expressive setup?